### ISSUE TITLE
Remove OpenG Dependencies?

### ISSUE THREAD

**Issue by AndrewHeim** [tannerblair/JSON-Support-for-LabVIEW#33]
I've come to not like large VI package-type dependencies.  They clutter projects and often get sucked up into projects unintentionally (creating other problems).

I'd love to see the OpenG packages distributed as a [Gpackage](https://gpackage.io/).  Until then... I see this listed on the Projects page - what's the chance that we remove those?

My immediate project sadly doesn't need array support (which is the case that relies on the OpenG VIs) so I'm likely going to just axe those cases and reduce functionality of the library for now.  I don't like it, but it's the shortest path for my case.

**Comment by AndrewHeim** [comment]
I tried.  After looking more at this... it doesn't seem likely unless we do a complete overhaul of the variant support.  I'll give up and close the issue.

**Comment by AndrewHeim** [comment]
Wow. I just realized in the commit history too that the OpenG dependencies were the answer to some of the earlier variant issues.  I didn't research this issue enough before I posted it.

To clarify for the future:  my team doesn't have a lot against OpenG in general.  But they do dislike the giant-package nature of the install.  Also, they dislike the way VI Package manger installs globally, since VIs have a way of finding themselves being sucked in other projects unintentionally.  We've moved towards GPM and it's per-project installation of dependencies - it tends to make each project cleaner and more self-documenting as well.
